static/src/javascripts-legacy/projects/common/modules/experiments/tests/editorial-email-variants.js

```diff
@@ -58,10 +58,10 @@ define([
         function runTheTest(emailListID, exampleUrl) {
           if (config.page.contentId === FLYERURL) {
             enhanceWebView(emailListID);
+            updateExampleUrl(exampleUrl);
           } else {
             updateNewslettersPage(emailListID);
           }
-          updateExampleUrl(exampleUrl);
```

Comment:
So, I've just realised/remembered that The Flyer does have a example on Newsletters, which is why it was outside the if-else block.

Maybe we need a check in updateExampleUrl() to see if the dom element exists...

I'll confess, I wrote this test assuming there would always be an example url